recharges/tests.py

```diff
@@ -321,20 +388,21 @@ def test_hotsocket_get_airtime_good(self):
             json.dumps(expected_response_good),
             status=200, content_type='application/json')
 
-        recharge_id = self.make_recharge(msisdn="+277244555", status=0)
+        recharge_id = self.make_recharge(msisdn="+27 7244555", status=0)
```

Comment:
It bugs me a little bit that this number is not the correct length - one character short.